apps/projects/app/store/init.js

```diff
@@ -28,13 +19,23 @@ export const initStore = vaultAddress => {
       // always return the state even unmodified
       return state
     },
-    [
-      // Always initialize the store with our own home-made event
-      of({ event: INITIALIZE_STORE }),
-      of({ event: INITIALIZE_VAULT }),
-      github(),
-      // handle vault events
-      vaultContract.events(),
-    ]
+    {
+      externals: [
+        // handle vault events
+        { contract: vaultContract },
+      ],
+      init: initState(vaultContract),
+    }
   )
 }
+
+const initState = (vaultContract) => async (cachedState = INITIAL_STATE) => {
+  let nextState = await initializeTokens(cachedState, vaultContract)
+  const github = await app.getCache('github').toPromise()
+  if (github && github.token) {
+    nextState.github = github
+    initializeGraphQLClient(github.token)
+  }
+
+  return nextState
+}
```

Comment:
All these changes going on into the projects store seem unrelated to the scope of the PR, they need to be separated and put into another request.

Reply:
We need these because the `api.cache` call went away and we need to use triggers instead. Maybe there's a simpler way to accomplish this; I followed the pattern started [in this PR](https://github.com/AutarkLabs/open-enterprise/pull/1160/files#diff-a62393c34897e310e6892a3aaf002ad2). Do you think it's important to make it more minimal?